### PR TITLE
move navbar content to burger menu on mobile

### PR DESCRIPTION
**Pull request by jdrouet**
Signed-off-by: Jérémie Drouet <jeremie.drouet@gmail.com>

### Proposed changes

Remove Guides / Product Manuals / Reference / Samples in the burger menu

The search field will be done in a separate PR because it implies more changes related to the search

### CODE REVIEW

**Comment by jdrouet** [comment]
Screen Shot 2020-05-20 at 09 47 36
Screen Shot 2020-05-20 at 09 47 40

**Comment by thaJeztah** [comment]
Do we want the logo / home link in the navigation bar? Something like:

hamburger

**Comment by thaJeztah** [comment]
Also keep wondering why we ever decided to "center" the links in the hamburger menu, and not left-align them as we have in the main navigation

That's not something introduced in this PR, but the screenshots made me think about that again 😂

**Comment by thaJeztah** [comment]
Otherwise, this looks good to me; it solves the issue that the first link in the hamburger-menu is hidden, and makes the navigation bar at the top less cluttered (giving more space for the actual content 👍)

<image width=250 src="https://user-images.githubusercontent.com/1804568/82428607-039d6600-9a8b-11ea-9c2c-a1b5da673932.png" />

**Comment by usha-mandya** [comment]
Thanks Jeremie. This looks good. 

Agree with @thaJeztah on the center alignment of the links on the other tabs. The links on the homepage look good with left alignment. 

image

**Comment by thaJeztah** [comment]
Oh, just noticed that the left side of the icon doesn't align with the left-side of the items in the menu (on the landing page they do); is that easy to adjust?

File: _includes/header.html [hunk 1]
```diff
@@ -2,10 +2,12 @@
     <!-- <div class="fan"></div> -->
```
[the remark this answers is not in the document]
Comment (jdrouet) [reply _includes/header.html:2]: I've no idea what it is made for 🤷

**Comment by jdrouet** [comment]
@thaJeztah should we align the text to the logo or the logo to the text?
If we align the logo to the text, I've to put the same padding to the burger button

**Comment by thaJeztah** [comment]
Hmm.. good question; the landing page seems to look ok (uses the same amount of whitespace on the left and right-hand side of the layout), so I think the other pages should possibly have a slightly smaller margin for the logo and the hamburger-menu?

**Comment by jdrouet** [comment]
@thaJeztah if we reduce the margin around the button and the logo, the body wouldn't be aligned

Screenshot 2020-05-25 at 16 17 27

**Comment by jdrouet** [comment]
Instead we can increase the margin on the menu
Screen Shot 2020-05-25 at 16 19 10

**Comment by jdrouet** [comment]
@thaJeztah we could move ahead with this PR and discuss about this with Rob

**Comment by usha-mandya** [comment]
@rgardziel Could you PTAL?

**Comment by thaJeztah** [comment]
Oh! I somehow didn't see the earlier comments.

>  if we reduce the margin around the button and the logo, the body wouldn't be aligned

Hm, I see the problem yes.

> Instead we can increase the margin on the menu

I'd say; pick that for now, and perhaps make the divider have the same width?

Screenshot 2020-05-26 at 09 35 10

**Comment by jdrouet** [comment]
@thaJeztah PTAL

File: _scss/_navigation.scss [hunk 2]
```diff
@@ -384,4 +389,9 @@ input:checked+.slider:before {
     margin: 2px 0 0;
 }
 
-
+.divider {
+    height: 1px;
+    margin: 9px 35px;
```
Comment (thaJeztah) [inline _scss/_navigation.scss:394]: Looks like changing this to 29px will make it the same width as the menu items;

```suggestion
    margin: 9px 29px;
```

Before:

Screenshot 2020-05-26 at 13 05 11

After:

Screenshot 2020-05-26 at 13 05 01

**Pull request merged**